src/input.rs

```diff
@@ -29,14 +29,20 @@ pub fn is_key_down(key_code: KeyCode) -> bool {
     context.keys_down.contains(&key_code)
 }
 
-/// Detect if the key is being pressed.
-/// Each call "is_key_down" call will consume a character from input queue.
-pub fn get_key_pressed() -> Option<char> {
+/// Return the last pressed char.
+pub fn get_last_char_pressed() -> Option<char> {
     let context = get_context();
 
     context.chars_pressed_queue.pop()
 }
 
+/// Return the last pressed key.
+pub fn get_last_key_pressed() -> Option<KeyCode> {
+    let context = get_context();
+
+    context.keys_pressed.iter().next().cloned()
```

Comment:
This will return not the last pressed key but a random one from the keys_pressed hashmap. 
And to be honest, it is fine for now, but maybe add a TODO?

Reply:
is this ok?
//TODO adjust function to return the last key

Reply:
I would do something like 

```
    // TODO: this will return a random key from keys_pressed HashMap instead of the last one, fix me later
    context.keys_pressed.iter().next().cloned() 
```